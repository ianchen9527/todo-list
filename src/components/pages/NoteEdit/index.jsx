import React from "react";
import styles from "./index.module.scss";
import NoteList from "../../../containers/NoteList";
import NoteIncreaser from "../../../containers/NoteIncreaser";
import NoteEditor from "../../../containers/NoteEditor";

const NoteEdit = () => {
  return (
    <div className={styles["wrapper"]}>
      <div className={styles["note-increaser-wrapper"]}>
        <NoteIncreaser />
      </div>
      <div className={styles["note-list-wrapper"]}>
        <NoteList />
      </div>
      <div className={styles["note-editor-wrapper"]}>
        <NoteEditor />
      </div>
    </div>
  );
};

export default NoteEdit;
