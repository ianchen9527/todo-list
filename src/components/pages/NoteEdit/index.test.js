import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NoteEdit from './index';

describe('Notes', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NoteEdit />);
  });

  it('should render without crashing', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
