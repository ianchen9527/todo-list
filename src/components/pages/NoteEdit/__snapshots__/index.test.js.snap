// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes should render without crashing 1`] = `
<div
  className="wrapper"
>
  <div
    className="note-increaser-wrapper"
  >
    <withRouter(Connect(NoteIncreaser)) />
  </div>
  <div
    className="note-list-wrapper"
  >
    <withRouter(Connect(withLifecycle(NoteList))) />
  </div>
  <div
    className="note-editor-wrapper"
  >
    <withRouter(Connect(withLifecycle(NoteEditor))) />
  </div>
</div>
`;
